doc/man/cockpit.conf.xml

```diff
@@ -149,6 +149,14 @@ ForwardedForHeader = X-Forwarded-For
           <code>/cockpit/</code> and <code>/cockpit+new/</code> are not.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><option>WebsocketRoot</option></term>
```

Comment:
We may not need this after all -- I tested talking to cockpit https over the /wss path, and that works. So let's not merge this part for now.